app/javascript/lib/honeycrisp.js

```diff
@@ -121,8 +121,8 @@ var followUpQuestion = (function () {
             });
         },
         update: function ($container) {
-            $container.find('.question-with-follow-up__follow-up input, .question-with-follow-up__follow-up select').attr('disabled', true);
-            $container.find('.question-with-follow-up__follow-up').hide();
+            $container.find('> .question-with-follow-up__follow-up input, .question-with-follow-up__follow-up select').attr('disabled', true);
+            $container.find('> .question-with-follow-up__follow-up').hide();
```

Comment:
just a note that this _used_ to exist in the previous followup implementation but it was removed in this PR https://github.com/codeforamerica/honeycrisp-gem/pull/312/files we might not have any instances of followup questions after none of the above checkboxes

Reply:
I'm not sure I see the direct descendent selector in that diff?

Reply:
Oops wrong one -- I'll post the right one after lunch

Reply:
https://github.com/codeforamerica/vita-min/commit/4f6da694d8e943320ac1536dbd6c19830933a499

I was mistaken- it seemed like we solved for this case a long time ago and it was a tax-specific code addition to honeycrisp.js that was never implemented in the gem, so when I copied it over we lost the existing fix